src/Model/Setting/AutomaticCollection.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Billogram\Model\Setting;
+
+use Billogram\Model\CreatableFromArray;
+
+class AutomaticCollection implements CreatableFromArray
```

Comment:
What is an AutomaticCollection? Why not AutomaticReminderCollection? Or ReminderCollection?